data/ecosystems/p/polygon.toml

```diff
@@ -5193,3 +5193,15 @@ url = "https://github.com/zmeghji/L2Token"
 
 [[repo]]
 url = "https://github.com/zmeghji/PolygonETB"
+
+[[repo]]
+url = "https://github.com/Plonky3/Plonky3"
+
+[[repo]]
+url = "https://github.com/valida-xyz/valida"
+
+[[repo]]
+url = "https://github.com/valida-xyz/valida-compiler"
+
+[[repo]]
+url = "https://github.com/valida-xyz/benchmarks"
```

Comment:
why should these repos by connected to Polygon? Would appreciate your helping validating these repos you added

Reply:
These repos are by the Polygon Zero team and will be used in conjunction with Polygon's 2.0 ZKEVM, read about polygon zero's involvement here in readme - https://github.com/Plonky3/Plonky3

Reply:
Ok, should we then just add them to the "Polygon Hermez" ecosystem toml file instead?